llvm/include/llvm/Support/Error.h

```diff
@@ -744,7 +744,7 @@ template <class T> class [[nodiscard]] Expected {
 /// @deprecated Use reportFatalInternalError() or reportFatalUsageError()
 /// instead.
 [[noreturn]] LLVM_ABI void report_fatal_error(Error Err,
-                                              bool gen_crash_diag = true);
+                                              bool GenCrashDiag = true);
```

Comment:
Unrelated rename

Reply:
Thanks. I'll remove anything that is just a tidy-up and not required.

Reply:
Done.